From 822dd046d7e22a8d01728200a003da230e4c6f7f Mon Sep 17 00:00:00 2001
From: Maxime Chevallier <maxime.chevallier@bootlin.com>
Date: Tue, 7 May 2019 17:35:55 +0200
Subject: [PATCH] dt-bindings: net: Fix a typo in the phy-mode list for
 ethernet bindings
Git-commit: 822dd046d7e22a8d01728200a003da230e4c6f7f
Patch-mainline: v5.1
References: bsc#1129770

The phy_mode "2000base-x" is actually supposed to be "1000base-x", even
though the commit title of the original patch says otherwise.

Fixes: 55601a880690 ("net: phy: Add 2000base-x, 2500base-x and rxaui modes")
Signed-off-by: Maxime Chevallier <maxime.chevallier@bootlin.com>
Reviewed-by: Andrew Lunn <andrew@lunn.ch>
Signed-off-by: David S. Miller <davem@davemloft.net>
Reviewed-by: Fabian Baumanis <fabian.baumanis@suse.com>
---
 Documentation/devicetree/bindings/net/ethernet.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/net/ethernet.txt b/Documentation/devicetree/bindings/net/ethernet.txt
index a68621580584..d45b5b56fa39 100644
--- a/Documentation/devicetree/bindings/net/ethernet.txt
+++ b/Documentation/devicetree/bindings/net/ethernet.txt
@@ -36,7 +36,7 @@ Documentation/devicetree/bindings/phy/phy-bindings.txt.
   * "smii"
   * "xgmii"
   * "trgmii"
-  * "2000base-x",
+  * "1000base-x",
   * "2500base-x",
   * "rxaui"
   * "xaui"
-- 
2.16.4

